extensions/amp-story-interactive/0.1/amp-story-interactive-quiz.js

```diff
@@ -65,8 +64,13 @@ export class AmpStoryInteractiveQuiz extends AmpStoryInteractive {
   constructor(element) {
     super(element, InteractiveType.QUIZ);
 
-    /** @private {!Array<string>} */
-    this.answerChoiceOptions_ = ['A', 'B', 'C', 'D'];
+    /** @private {!Array<../../../src/localized-strings.LocalizedStringId>} */
+    this.answerChoiceOptions_ = [
```

Comment:
Nit: let's update this variable name to reflect what's inside it. I know you mutate it later, which makes it even more confusing because the type is no longer correct

Reply:
That's true, technically the type is always a string but pr-check required it to be this type in the JSDoc. I'll change the name so that it shows it can be the localized values

Reply:
If having two variables (one for the localizedIds and one for the strings) make things easier to understand and more accurate, we should go for it

Reply:
Alternately, we could construct the localizedIds array inline, right before sending them through the localization service.

Reply:
I don't feel the need to have two variables, since they just clutter the component (many components have so many variables that it's hard to understand what they do). Because of the type inconsistency, constructing the array in-line makes the most sense, as the LocalizedStringId keys won't be used anywhere else. Sent the commit with it